mobile/test/java/org/chromium/net/testing/CronetTestRule.java

```diff
@@ -72,30 +69,20 @@ public static class CronetTestFramework {
     public ExperimentalCronetEngine.Builder mBuilder;
 
     private Context mContext;
-    private boolean mIsTestingJavaImpl;
 
-    private CronetTestFramework(Context context, boolean isTestingJavaImpl) {
```

Comment:
What the story with java impl vs native impl? (I suspect this is wrapped up with why we're able to remove those annotations?)

Reply:
Answered in another comment.
In the testing field, since we no longer have java fallback impl, its corresponding tests need to be removed.